### DISCUSSION TITLE
How to make vweb based application listen to all interface ?

### DISCUSSION THREAD

**Discussion by freezmeinster** [vlang/v#10966] [General]
I am now building an experimental FreeBSD Virtualization management interface that use Vweb. But now i have a problem, my application only listening to localhost. 

Based on this code https://github.com/vlang/v/blob/063ffe36d981a294b00251dd4c7b59df44eb467b/vlib/vweb/vweb.v#L291
it's seem that Vweb hard coded to listening only  on localhost. 
It's is pointless for me to deploy extra reverse proxy like Nginx in front of application. 

So the question is : 

1. Is my understanding about vweb listening only on localhost wrong ?
2. How to make vweb listening to all interface or IP available ?

Thank you in advance

**Comment by mindplay-dk** [comment]
I don't know much about it, I just manually followed your call back to this line:

https://github.com/vlang/v/blob/fde934cd93e7dea8b6654bd21b8d26176bc98b83/vlib/net/address.v#L164-L167

I should be listening on `net.addr_ip6_any`, which is [filled with zeroes](https://github.com/vlang/v/blob/fde934cd93e7dea8b6654bd21b8d26176bc98b83/vlib/net/address.v#L13) - I couldn't tell you if that's the correct way to do it or not...
